shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/converter/segment/projection/impl/AggregationProjectionConverter.java

```diff
@@ -86,4 +137,15 @@ private SqlAggFunction convertOperator(final String operator) {
         Preconditions.checkState(REGISTRY.containsKey(operator), "Unsupported SQL operator: `%s`", operator);
         return REGISTRY.get(operator);
     }
+    
+    /**
+     * Is sqlBasicCall as operator aggregation type.
```

Comment:
@LeeGuoPing `Judge whether sqlBasicCall  is as operator aggregation type or not.` may be better.